Tests/Packet++Test/Tests/SomeIpTests.cpp

```diff
@@ -148,3 +149,161 @@ PTF_TEST_CASE(SomeIpCreationTest)
 	delete [] buffer1;
 	delete [] buffer2;
 }
+
+class SomeIpTeardown
```

Comment:
It's great that you added it!

I think we should also use it in `SomeIpParsingTest` (and move `SomeIpTeardown` to the top of the file)

Reply:
That's a good point!
Maybe we can do something like this:
- Modify `SomeIpTeardown()` c'tor to not take any argument (and then we don't need `m_SomeIpPort`)
- Modify `~SomeIpTeardown()` d'tor to use `removeAllSomeIpPorts()` - so it will clean up all the ports in the end of the test
- Add another test case for testing `addSomeIpPort()`, `removeSomeIpPort()`, `removeAllSomeIpPorts()` and `isSomeIpPort()`, it can do something like this:
 ```cpp
PTF_TEST_CASE(SomeIpPortTest)
{
	SomeIpTeardown tearDown;

	pcpp::SomeIpLayer::addSomeIpPort(1234);
	PTF_ASSERT_TRUE(pcpp::SomeIpLayer::isSomeIpPort(1234));

	pcpp::SomeIpLayer::removeSomeIpPort(1234);
	PTF_ASSERT_FALSE(pcpp::SomeIpLayer::isSomeIpPort(1234));

	pcpp::SomeIpLayer::addSomeIpPort(1235);
	pcpp::SomeIpLayer::addSomeIpPort(1236);
	PTF_ASSERT_TRUE(pcpp::SomeIpLayer::isSomeIpPort(1235));
	PTF_ASSERT_TRUE(pcpp::SomeIpLayer::isSomeIpPort(1236));

	pcpp::SomeIpLayer::removeAllSomeIpPorts();
	PTF_ASSERT_FALSE(pcpp::SomeIpLayer::isSomeIpPort(1235));
	PTF_ASSERT_FALSE(pcpp::SomeIpLayer::isSomeIpPort(1236));
}
```

Reply:
ok, for some reason `SomeIpTeardown tearDown();` did not work, but now it should be fine. Can we bring this to the dev branch?